File: src/main/java/ch/qligier/jetbrains/plugin/fss/igini/language/psi/IniTokenSets.java
// Copyright 2024 Quentin Ligier. Use of this source code is governed by the MIT license.

package ch.qligier.jetbrains.plugin.fss.igini.language.psi;

import com.intellij.psi.tree.TokenSet;

/**
 * jetbrains-plugin-fss
 *
 * @author Quentin Ligier
 **/
public interface IniTokenSets {

    TokenSet COMMENTS = TokenSet.create(IniTypes.COMMENT);
}

File: src/main/kotlin/ch/qligier/jetbrains/plugin/fss/igini/language/psi/IniTokenSets.kt
/*
 * Copyright 2025 Quentin Ligier. Use of this source code is governed by the MIT license.
 */

package ch.qligier.jetbrains.plugin.fss.igini.language.psi

import com.intellij.psi.tree.TokenSet

interface IniTokenSets {
    companion object {
        val COMMENTS: TokenSet = TokenSet.create(IniTypes.COMMENT)
    }
}
